staging/src/k8s.io/client-go/go.mod

```diff
@@ -0,0 +1,37 @@
+module k8s.io/client-go
+
+require (
+	k8s.io/api v1.12.0
+	k8s.io/apimachinery v1.12.0
+)
+
+replace (
```

Comment:
if I understand correctly, we only have to remove this block during publishing of the staging repos. Then the file is valid in a normal non-monorepo world. 👍

Reply:
Ok, even better (= simpler).